### PR TITLE
Add Stripe connection for streamlined Donate block

### PR DESCRIPTION
**Pull request by adekbadek**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Follow instructions in https://github.com/Automattic/newspack-blocks/pull/784

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: assets/wizards/readerRevenue/index.js [hunk 1]
```diff
@@ -203,29 +214,10 @@ class ReaderRevenueWizard extends Component {
...
-					label: __( 'Platform', 'newspack' ),
-					path: '/',
-					exact: true,
-				},
+				...platformAgnosticFields,
 			];
 		} else if ( NRH === platform ) {
-			return [
-				{
-					label: __( 'Settings', 'newspack' ),
```
Comment (dkoo) [inline assets/wizards/readerRevenue/index.js:219]: Don't we still need to be able to access the "Settings" tab in the dashboard to enter `NRH Organization ID` and `NRH Salesforce Campaign ID` options, if NRH is the platform but _not_ using the streamlined donation block?

**Review by dkoo — CHANGES_REQUESTED** [review]
As explained [here](https://github.com/Automattic/newspack-blocks/pull/784#pullrequestreview-697462426), I couldn't fully test this feature due to a JS error I encountered on the front-end. I do have one blocking question below which I think we need to address in the meantime, and two non-blocking questions about using WooCommerce helper functions.

File: includes/util.php [hunk 2]
```diff
@@ -36,3 +36,365 @@ function newspack_clean( $var ) {
 function newspack_string_to_bool( $string ) {
 	return is_bool( $string ) ? $string : ( 'yes' === $string || 1 === $string || 'true' === $string || '1' === $string );
 }
+
+/**
+ * Currencies options, copied from WooCommerce.
+ * https://github.com/woocommerce/woocommerce/blob/master/includes/wc-core-functions.php
+ */
+function newspack_get_currencies_options() {
```
[the remark this answers is not in the document]
Comment (adekbadek) [reply includes/util.php:44]: If the platform is NRH, WC is not a dependency.

File: includes/util.php [hunk 3]
```diff
@@ -36,3 +36,365 @@ function newspack_clean( $var ) {
...
+	return $currency_options;
+}
+
+/**
+ * Get symbol of a currency.
+ * https://github.com/woocommerce/woocommerce/blob/master/includes/wc-core-functions.php#L845
+ *
+ * @param string $currency_code Currency code.
+ */
+function newspack_get_currency_symbol( $currency_code ) {
```
[the remark this answers is not in the document]
Comment (adekbadek) [reply includes/util.php:231]: If the platform is NRH, WC is not a dependency.

File: assets/wizards/readerRevenue/index.js (hunk 1 above)
Comment (adekbadek) [reply assets/wizards/readerRevenue/index.js:219]: Whoops, that's true. Re-added and changed tab title to "NRH Settings" for clarity in 5b162874ca54f500da929883aeb32f6860d4bb99

File: includes/util.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (dkoo) [reply includes/util.php:44]: Good point, thanks for clarifying!

File: assets/wizards/readerRevenue/index.js (hunk 1 above)
Comment (dkoo) [reply assets/wizards/readerRevenue/index.js:219]: This looks great! But now I want the "Stripe" tab to be labelled "Stripe Settings" for consistency? 😬

**Review by dkoo — APPROVED** [review]
https://github.com/Automattic/newspack-plugin/pull/1029/files#r663251207, what do you think?

**Review by dkoo — CHANGES_REQUESTED** [review]
Meant to request changes, sorry

File: assets/wizards/readerRevenue/index.js (hunk 1 above)
Comment (adekbadek) [reply assets/wizards/readerRevenue/index.js:219]: Yeah, that makes sense - added inf2abcba346b07b7ef716da71b5bfb2726b49ff37

**Review by dkoo — APPROVED** [review]

**Pull request merged**